src/main/java/com/acrolinx/client/demo/SdkDemo.java

```diff
@@ -48,45 +52,44 @@ public class SdkDemo {
   private static final Logger LOGGER = LoggerFactory.getLogger(SdkDemo.class);
 
   public static void main(String[] args)
-      throws URISyntaxException, InterruptedException, AcrolinxException {
-    AcrolinxEndpoint acrolinxEndpoint =
-        new AcrolinxEndpoint(
-            new URI(ACROLINX_URL), CLIENT_SIGNATURE, CLIENT_VERSION, CLIENT_LOCALE);
-    AccessToken accessToken = signInInteractive(acrolinxEndpoint);
-    GuidanceProfile guidanceProfile = getGuidanceProfiles(acrolinxEndpoint, accessToken);
-    CheckOptions checkOptions = createCheckOptions(guidanceProfile);
-
-    CheckResult checkResult =
-        acrolinxEndpoint.check(
-            accessToken,
-            createCheckRequest(checkOptions),
-            progress ->
-                LOGGER.info("Progress: {}% ({})", progress.getPercent(), progress.getMessage()));
+      throws URISyntaxException, InterruptedException, AcrolinxException, IOException {
 
-    LOGGER.info("Score: {}", checkResult.getQuality().getScore());
-    LOGGER.info("Status: {}", checkResult.getQuality().getStatus());
-    LOGGER.info("Scorecard: {}", checkResult.getReport(ReportType.scorecard).getLink());
+    if (args.length > 0 && args[0].equals("docx")) {
```

Comment:
I am not happy with this logic, passing any parameter to switch between these two modes. My suggestion would be to either
a) pass the file name of the file that should be checked (which opens the question of error handling, what if the file does note exist, is of a different type/encoding, etc...)
or
b) we can create two separate Java classes for each scenario, each having a public static main method. To avoid redundant code, we should not duplicate them but extract common methods etc into a shared class. Then none of the two main methods needs to take any arguments and we can simply document in the README how to run these two different scenarios.

What do you think?